Misc/NEWS.d/next/Library/2023-12-17-10-22-55.gh-issue-112182.jLWGlr.rst

```diff
@@ -1,2 +1,3 @@
-:meth:`asyncio.futures.Future.set_exception()` now silently transforms
-``StopIteration`` into ``RuntimeError``. Patch contributed by Jamie Phan.
+:meth:`asyncio.futures.Future.set_exception()` now transforms ``StopIteration``
+into ``RuntimeError`` instead of hanging or other misbehavior. Patch
```

Comment:
```suggestion
:meth:`asyncio.futures.Future.set_exception()` now transforms :exc:`StopIteration`
into :exc:`RuntimeError` instead of hanging or other misbehavior. Patch
```